sdk/tracing/src/main/java/io/opentelemetry/sdk/trace/SpanProcessor.java

```diff
@@ -8,12 +8,37 @@
 import io.opentelemetry.api.trace.Span;
 import io.opentelemetry.context.Context;
 import io.opentelemetry.sdk.common.CompletableResultCode;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
 
 /**
  * SpanProcessor is the interface {@code TracerSdk} uses to allow synchronous hooks for when a
  * {@code Span} is started or when a {@code Span} is ended.
  */
 public interface SpanProcessor {
+
+  /**
+   * Returns a {@link SpanProcessor} which simply delegates all processing to the {@code processors}
+   * in order.
+   */
+  static SpanProcessor delegating(SpanProcessor... processors) {
+    return delegating(Arrays.asList(processors));
+  }
+
+  /**
+   * Returns a {@link SpanProcessor} which simply delegates all processing to the {@code processors}
+   * in order.
+   */
+  @SuppressWarnings("deprecation")
+  static SpanProcessor delegating(Iterable<SpanProcessor> processors) {
```

Comment:
`allOf()` ?

Reply:
or `ofAll()` ?  😆

Reply:
`composite` or `multi`, `exportEach`

Reply:
I like `composite` or `composedOf`

Reply:
`composite` or `multi` 😄

Reply:
composite has votes from everyone including me so there we go :)